Sources/StreamChatUI/ChatChannelList/ChatChannelListVC+SwiftUI.swift

```diff
@@ -6,7 +6,7 @@ import StreamChat
 import SwiftUI
 
 @available(iOS 13.0, *)
-/// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
+// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
```

Comment:
Hm, it's not an inline comment, is it? why lint is not happy with that? 🤔

Reply:
Yeah, it's because it's a typealias 😭